src/sass/patterns/_lists.scss

```diff
@@ -27,3 +27,8 @@ dl {
 dd {
   margin-inline-start: var(--description-margin-inline, var(--gutter));
 }
+
+.list-item-heading {
+  font-weight: bold;
+  font-size: var(--small);
```

Comment:
```suggestion
  font-size: var(--small);
  font-weight: bold;
```

:)

Reply:
fixed in https://github.com/oddbird/oddcontrast/pull/223/commits/7e27f5eca78df035e0d1145ae0b81dd0346b27c2 :)